app/models/commits_service.rb

```diff
@@ -0,0 +1,8 @@
+class CommitsService
+
+  def self.find_commits(username, repository)
+    response = Faraday.get("https://api.github.com/repos/#{username}/#{repository}/commits?client_id=#{ENV['client_id']}&client_secret=#{ENV['client_secret']}")
```

Comment:
@case-eee So i just realized why Im doing it the long way. It's for testing purposes. If I try to send the request with an access token it gets a 'bad credentials' error. I tried adding the access_token to my hidden apllications.yml file and manually changing every factory created user to have my access_token. I couldn't get it to work. Maybe this is something I can go over with you whenever it is we get around to refactoring some of this code.